deepcell/utils/train_utils.py

```diff
@@ -54,16 +54,26 @@ def get_callbacks(model_path,
         tensorboard_log_dir (str): log directory for tensorboard.
         monitor (str): quantity to monitor.
         verbose (int): verbosity mode, 0 or 1.
+        manual_save (int or None): optional parameter specifying period
+            to save model at, irrespective of validation accuracy improvements
 
     Returns:
         list: a list of callbacks to be passed to model.fit()
     """
-    cbs = [
-        callbacks.ModelCheckpoint(
-            model_path, monitor=monitor,
-            save_best_only=True, verbose=verbose,
-            save_weights_only=save_weights_only),
-    ]
+    if manual_save is None:
```

Comment:
Instead, let's pop the value out of kwargs and default it to the default value in the model checkpoint just to reduce duplicated code.

Reply:
Okay @willgraf, so pass manual_save (or save_freq, see my comment above) as part of kwargs. Then get that value from kwargs, rather than defaulting it to None in the training call. We'll still need to duplicate the cbs initialization, right? Given that monitor, save_best_only, and save_freq will be different? Or do you have an idea of how we could set default values for those earlier on that would just get popped out?